test/data/test_builtin_datasets.py

```diff
@@ -65,7 +65,7 @@ def test_wikitext2(self):
         # Add test for the subset of the standard datasets
         train_dataset, test_dataset = torchtext.experimental.datasets.raw.WikiText2(data_select=('train', 'test'))
         self._helper_test_func(len(train_dataset), 36718, next(iter(train_dataset)), ' \n')
-        self._helper_test_func(len(test_dataset), 36718, next(iter(test_dataset)), ' \n')
+        self._helper_test_func(len(test_dataset), 4358, next(iter(test_dataset)), ' \n')
```

Comment:
We just need to spread this to the other datasets as well and then this is good to go